pkg/networkutils/network.go

```diff
@@ -622,7 +618,7 @@ func setupENINetwork(eniIP string, eniMAC string, eniTable int, eniSubnetCIDR st
 						log.Errorf("Failed to add route %s/0 via %s table %d",
 							r.Dst.IP.String(), gw.String(), eniTable)
 						return errors.Wrapf(err,
-							"eni network setup: failed unable to add route %s/0 via %s table %d",
+							"ENI network setup: failed unable to add route %s/0 via %s table %d",
```

Comment:
This or a different PR (also to match the log.Errorf)
s/failed unable to add route/failed to add route/

Reply:
Thanks, good catch.